descriptions/api.github.com/api.github.com.json

```diff
@@ -51850,6 +51850,109 @@
+        }
+      },
+      "delete": {
+        "summary": "Delete a package for a user",
+        "description": "Deletes an entire package for a user. You cannot delete a public package if any version of the package has more than 5,000 downloads. In this scenario, contact GitHub support for further assistance.\n\nTo use this endpoint, you must authenticate using an access token with the `packages:read` and `packages:delete` scopes. In addition:\n- If `package_type` is not `container`, your token must also include the `repo` scope.\n- If `package_type` is `container`, you must also have admin permissions to the container you want to delete.",
+        "tags": [
+          "packages"
+        ],
+        "operationId": "packages/delete-package-for-user",
+        "externalDocs": {
+          "description": "API method documentation",
+          "url": "https://docs.github.com/rest/reference/packages#delete-a-package-for-a-user"
+        },
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/package-type"
+          },
+          {
+            "$ref": "#/components/parameters/package-name"
+          },
+          {
+            "$ref": "#/components/parameters/username"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Response"
+          },
+          "404": {
+            "$ref": "#/components/responses/not_found"
+          },
+          "403": {
+            "$ref": "#/components/responses/forbidden"
+          },
+          "401": {
+            "$ref": "#/components/responses/requires_authentication"
+          }
+        },
+        "x-github": {
+          "githubCloudOnly": false,
+          "enabledForGitHubApps": false,
+          "previews": [
+
+          ],
+          "category": "packages",
+          "subcategory": null
+        }
+      }
+    },
+    "/users/{username}/packages/{package_type}/{package_name}/restore": {
+      "post": {
+        "summary": "Restore a package for a user",
+        "description": "Restores an entire package for a user.\n\nYou can restore a deleted package under the following conditions:\n  - The package was deleted within the last 30 days.\n  - The same package namespace and version is still available and not reused for a new package. If the same package namespace is not available, you will not be able to restore your package. In this scenario, to restore the deleted package, you must delete the new package that uses the deleted package's namespace first.\n\nTo use this endpoint, you must authenticate using an access token with the `packages:read` and `packages:write` scopes. In addition:\n- If `package_type` is not `container`, your token must also include the `repo` scope.\n- If `package_type` is `container`, you must also have admin permissions to the container that you want to restore.",
+        "tags": [
+          "packages"
+        ],
+        "operationId": "packages/restore-package-for-user",
```

Comment:
please change operationId to `packages/delete-package-for-authenticated-user` to be consistent with the other `/user/*` operations

Reply:
sorry these `operationId` is actually correct, it's a `/users/{username}` path, so the `-for-user` suffix is correct